shared/video-sdk/develop/product-workflow/project-implementation/flutter.mdx

```diff
@@ -69,44 +69,102 @@ To implement these features in your <Vpl k="CLIENT" />, take the following steps
-        ChannelMediaOptions options = ChannelMediaOptions(
-            publishScreenTrack: _isScreenShared,
-            publishCameraTrack: !_isScreenShared,
-        );
+    Depending on your target platform, you call a suitable <Vpd k="SDK" /> method to start screen sharing when the button is pressed. Copy the code for your target platform and add it to `shareScreen()` after `// Start screen sharing`:
 
-        agoraEngine.updateChannelMediaOptions(options);
-    }
-    ```
+    * **Windows and macOS**
+
+        On multi-window system, you obtain a list of available windows and displays and choose the source to share.
 
-    ---
-    **Note**
+        ```dart
+        // Get the list of available windows and displays.
+        List<ScreenCaptureSourceInfo> screenCaptureSourceList;
+        screenCaptureSourceList = await agoraEngine.getScreenCaptureSources(
+            thumbSize: const SIZE(width: 360, height: 240),
+            iconSize: const SIZE(width: 360, height: 240),
+            includeScreen: false);
 
-    The Flutter screen sharing methods used in this example currently work only on the Windows platform. You need a custom implementation to enable screen sharing on other platforms.
-    ---
+        // In a real-life app, you list the sources and let the user choose.
+        // For this demo, get the Id of the last item in the list.
+        int? sourceId = screenCaptureSourceList.last.sourceId ?? 0;
+
+        // Share the entire screen or a particular window.
+        if (screenCaptureSourceList.last.type == ScreenCaptureSourceType.screencapturesourcetypeScreen) {
+            // The source is a screen
+            agoraEngine.startScreenCaptureByDisplayId(
+                displayId: sourceId,
+                regionRect: const Rectangle(),
+                captureParams: const ScreenCaptureParameters(
+                captureMouseCursor: true,
+                frameRate: 30,
+                ));
+        } else { 
+            // The source is a window
+            agoraEngine.startScreenCaptureByWindowId(
+                windowId: sourceId,
+                regionRect: const Rectangle(),
+                captureParams: const ScreenCaptureParameters(
+                captureMouseCursor: true,
+                frameRate: 30,
+                ));
+        }        
+        ```
+
+    * **iOS and Android**
+
+        On iOS and Android, you share the entire screen using the following method:
+
+        ```dart
+        agoraEngine.startScreenCapture(const ScreenCaptureParameters2(
+            captureAudio: true,
+            audioParams: ScreenAudioParameters(
+                sampleRate: 16000, channels: 2, captureSignalVolume: 100),
+            captureVideo: true,
+            videoParams: ScreenVideoParameters(
+                dimensions: VideoDimensions(height: 1280, width: 720),
+                frameRate: 15,
+                bitrate: 600)));  
+        ```
+        
+        :warning: **Note**: 
```

Comment:
The iOS stuff should be part of the prerequisites. Can you please update the iOS prereqs for flutter to include the platform version. 


And the Android bit as a step in the projet setup please.

Reply:
Done